src/test/java/de/zalando/OrderMonetaryAmountTransformer.java

```diff
@@ -1,5 +1,5 @@
 
-package org.zalando.sprocwrapper.example.transformer;
+package de.zalando;
```

Comment:
Why did you change the package here?

Reply:
Thank you for reviewing. I think we need to implement this workaround in order to not break clients. We can convert namespace variable to array. In this case we will expect from the clients to pass old namespace in **global.value.transformer.search.namespace**. And as a client of this library I was not aware of global.value.transformer.search.namespace until analyze the source code. 
For long story short, let's  make **global.value.transformer.search.namespace** an array and put some information in README.md ? What do think ? If you are ok with that, I will change the PR

Reply:
Yes, sounds good!